src/conditionals/import-tool-selected-conditional.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Yoast\WP\SEO\Conditionals;
+
+/**
+ * Conditional that is only met when current page is not a specific tool's page.
+ */
+class Import_Tool_Selected_Conditional implements Conditional {
```

Comment:
I think we should add a few basic unit tests for this class to ensure code coverage is maintained.